.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -76,28 +80,37 @@ Update versions in the following dockerfiles
 - [ ] `./dataland-api-key-manager/Dockerfile`
 - [ ] `./dataland-api-key-manager/DockerfileBase`
 - [ ] `./dataland-api-key-manager/DockerfileTest`
+- [ ] `./dataland-automated-qa-service/Dockerfile`
+- [ ] `./dataland-automated-qa-service/DockerfileBase`
+- [ ] `./dataland-automated-qa-service/DockerfileTest`
 - [ ] `./dataland-backend/Dockerfile`
 - [ ] `./dataland-backend/DockerfileBase`
 - [ ] `./dataland-backend/DockerfileTest`
+- [ ] `./dataland-batch-manager/Dockerfile`
+- [ ] `./dataland-batch-manager/DockerfileBase`
+- [ ] `./dataland-community-manager/Dockerfile`
+- [ ] `./dataland-community-manager/DockerfileBase`
+- [ ] `./dataland-community-manager/DockerfileTest`
```

Comment:
Why were these added Dockerfiles not listed before? Is this just for the sake of completeness?

Reply:
Yes